AuroraStudiosDigipen/Borealis
# Project building

Added Project building and revised scene management for purpose of project framework

How to test:

Scene Management:
In the File Tab, click new, and write a project name, this will create a project directory at the directory you were in
Copy and paste assets from level editor folder into the new project folder to bring all the files there
Drag imGuizmo.sc and example.sc from scenes folder into the viewport => Scene hierarchy should show all these saved scenes
Right click the grayed out scene names in the scene hierarchy and click load, it should be able to switch between the scenes

Building Project:
Switch to the imGuizmo scene and File Tab -> Save OR ctrl+S
File Tab -> Build Project -> Write the name for the build, it will create a folder and inside the folder there will be an exe. You should be able to run the exe and it should show what was left on the viewport in level editor, which is the boxes and the circle
